concepticondata/concepticon.tsv

```diff
@@ -2346,7 +2346,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2345	LEPORID (RABBIT OR HARE)	Animals	Mammal of the family hares and rabbits (Leporidae) with long ears, short tail and hindlegs which are shorter than the forelegs and permit running quickly.	Person/Thing	
 2346	CRESTED MYNA	Animals	A species of starling native to southeastern China and Indochina.	Person/Thing	
 2347	YELLOW CROAKER	Animals	A species of croaker native to the western Pacific, generally in temperate waters such as the East China Sea and the Yellow Sea.	Person/Thing	
-2348	CRAB	Animals	"Decapod crustaceans of the infraorder Brachyura, which typically have a very short projecting ""tail"" (abdomen) (Greek: βραχύς / brachys = short, οὐρά / οura = tail), usually entirely hidden under the thorax."	Person/Thing	
+2348	CRAB	Animals	Decapod crustaceans of the infraorder Brachyura, which typically have a very short projecting "tail" (abdomen) (Greek: βραχύς / brachys = short, οὐρά / οura = tail), usually entirely hidden under the thorax.	Person/Thing	
```

Comment:
Office is going to mess up that one every time - can't we just escape the quotes?

Reply:
I'll open a separate issue to discuss the "quotes". Because some descriptions include "&quote".